diff --git a/SELECT *.sql b/SELECT *.sql
--- SELECT *.sql
+++ SELECT *.sql
@@ -1,4 +1,37 @@
-SELECT *
-FROM Vendors
-WHERE VendorName LIKE 'A%' OR VendorName LIKE 'B%'
-ORDER BY VendorName;
\ No newline at end of file
+-- quiz 6
+-- question 1
+
+/*
+Write a script that declares and sets a variable that’s equal to the count of all rows in the Invoices table that have a
+balance due that’s greater than $5,000.00. Then, the script should display a message that looks like this: “2
+invoices exceed $5,000. */
+
+DECLARE @InvoiceCount INT;
+SELECT @InvoiceCount = COUNT(*)
+FROM Invoices
+WHERE InvoiceTotal - PaymentTotal - CreditTotal > 5000.00;
+SELECT CAST(@InvoiceCount, ' Invoices exceed $5,000.') AS MESSAGE;
+
+
+
+/* question 2 Write a script that uses variables to get (1) the count of all the invoices in the Invoices table that have a balance due
+and (2) the sum of the balances due for all those invoices. If that total balance due is greater than or equal to
+$10,000, the script should return a result set consisting of VendorName, InvoiceNumber, InvoiceDueDate, and
+Balance for each invoice with a balance due, sorted with the oldest due date first. The script should also display a
+message like this:
+Number of unpaid invoices is 11.
+Total balance due is $32,020.42.
+Otherwise, the script should display: “Total balance due is less than $10,000.”
+Stored Procedures
+3. Create a stored procedure named spVendorsWithoutInvoices that accepts @VendorName as a required parameter.
+The procedure returns a result set consisting of VendorID and VendorName for each vendor that has no invoices,
+sorted with VendorName. Call the procedure with vendor names that contain the word ‘service’ or ‘services’. (5
+row(s) affected)
+4. Create a stored procedure named spVendorStateInvTotal that accepts @VendorState as an optional parameter and
+@SumInvoiceTotal (i.e. the sum of all the invoices) as an output parameter. Then, call the procedure as follows and
+print out the value of the output parameter:
+a. Without providing @VendorState. (214290.51)
+b. With @VendorState = ‘tx’. (2154.42)
+c. With @VendorState = ‘t%’. (6532.44) */
+
+
